### PR TITLE
Update wireshark.pm to fix poo#16638

### PR DESCRIPTION
**Pull request by chuchingkai**
Verification test: http://147.2.212.103/tests/569

see also: https://progress.opensuse.org/issues/16638

### CODE REVIEW

File: tests/x11/wireshark.pm [hunk 1]
```diff
@@ -167,6 +169,7 @@ sub run() {
         assert_screen "wireshark-preferences-columns-protocol-displayed-selected";
         assert_and_click "wireshark-preferences-apply";
     }
+    wait_still_screen 3;
```
Comment (mitiao) [inline tests/x11/wireshark.pm]: Should this wait_still_screen be quoted in above 'if' block?

Comment (chuchingkai) [inline tests/x11/wireshark.pm]: fixed

**Comment by plusky** [comment]
Major version update is still in queue - see https://fate.suse.com/321770, it will requires significant changes to wireshark test anyways (see poo#17954)

**Comment by okurz** [comment]
@SergioAtSUSE I would be interested in how the test could look like with your changes. You can propose another PR based on this
@plusky this is why @chuchingkai closed his previous PR. But from the fate entry I read it as "in about one year from now" as an estimation. We should definitely not try to keep a failed test around for that long. I mean, why not apply a simple fix to make the test work again?

**Comment by plusky** [comment]
@okurz true - but its definitely not one year from now

**Comment by plusky** [comment]
@SergioAtSUSE @ostruzina is now new mainteiner of this test so maybe you should get in touch with her in regard of suggested changes.

**Pull request merged**